scripts/core/editor3/actions/spellchecker.jsx

```diff
@@ -0,0 +1,13 @@
+export function replaceWord(word) {
+    return {
+        type: 'REPLACE_WORD',
```

Comment:
Based on the editor approach, should we do the same here? `SPELLCHECKER_REPLACE_WORD` ?